src/test/java/com/notably/logic/parser/suggestion/DeleteSuggestionCommandParserTest.java

```diff
@@ -16,73 +16,28 @@
 import com.notably.logic.commands.suggestion.SuggestionCommand;
 import com.notably.logic.correction.AbsolutePathCorrectionEngine;
 import com.notably.logic.correction.CorrectionEngine;
+import com.notably.logic.suggestion.SuggestionTestUtil;
 import com.notably.model.Model;
-import com.notably.model.ModelManager;
-import com.notably.model.block.Block;
-import com.notably.model.block.BlockImpl;
-import com.notably.model.block.BlockModel;
-import com.notably.model.block.BlockModelImpl;
-import com.notably.model.block.Title;
 import com.notably.model.suggestion.SuggestionItem;
-import com.notably.model.suggestion.SuggestionItemImpl;
-import com.notably.model.suggestion.SuggestionModel;
-import com.notably.model.suggestion.SuggestionModelImpl;
-import com.notably.model.viewstate.ViewStateModel;
-import com.notably.model.viewstate.ViewStateModelImpl;
 
 public class DeleteSuggestionCommandParserTest {
-    private static AbsolutePath toRoot;
-    private static AbsolutePath toCs2103;
-    private static AbsolutePath toCs3230;
-    private static AbsolutePath toCs2103Week1;
-    private static AbsolutePath toCs2103Week2;
-    private static AbsolutePath toCs2103Week3;
-    private static AbsolutePath toCs2103Week1Lecture;
-    private static Model model;
+    private static AbsolutePath toCs2103t;
     private static DeleteSuggestionCommandParser deleteSuggestionCommandParser;
+    private static Model model;
 
+    private static final String RELATIVE_PATH_TO_CS2103T = "CS2103T";
```

Comment:
Isn't it better if you can move this to your `SuggestionTestUtil` class? It'll make it consistent with your `toCs2013t` `AbsolutePath`